packages/testlab/README.md

```diff
@@ -16,6 +25,41 @@ expect({key:'value'}).to.deepEqual({key: 'value'});
 expect.exists(1);
 ```
 
-## `sinon`
+```ts
+it('calls the original function', function () {
+    var callback = sinon.spy();
+    var proxy = once(callback);
+
+    proxy();
+
+    assert(callback.called);
```

Comment:
1) LoopBack apps should use `expect`, therefore our code examples should do the same.
2) I don't think it's necessary to show all helpers in the "Basic use" section. Let's stick to a single example that we had in "Example usage".